### PR TITLE
Fix typo in MediaDevices.getDisplayMedia()

### PR DESCRIPTION
**Pull request by watfordsuzy**
Small typo fix in `MediaDevices.getDisplayMedia()`.

### CODE REVIEW

File: files/en-us/web/api/mediadevices/getdisplaymedia/index.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h3 id="Return_value">Return value</h3>
 <div class="notecard note">
   <p><strong>Note:</strong> Browser support for audio tracks varies, both in terms of
     whether or not they're supported at all by the media recorder and in terms of the
-    which audio source or sourcoes are supported. Check the {{anch("Browser
+    which audio source or sources are supported. Check the {{anch("Browser
```
Comment (teoli2003) [inline files/en-us/web/api/mediadevices/getdisplaymedia/index.html:58]: Thanks for the typo, but I  think the sentence is not grammarly correct.

What about:
```suggestion
     audio sources supported. Check the {{anch("Browser
```
?

Comment (watfordsuzy) [inline files/en-us/web/api/mediadevices/getdisplaymedia/index.html:58]:
```suggestion
    audio sources supported. Check the {{anch("Browser
```

Comment (watfordsuzy) [reply files/en-us/web/api/mediadevices/getdisplaymedia/index.html]: Funny what my eyes miss :)

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here!

**Comment by watfordsuzy** [comment]
Happy to help my favorite JS in the browser reference.

**Pull request merged**